content/docs/practices/histograms.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Histograms and summaries
+sort_rank: 4
+---
+
+# Histograms and summaries
+
+Histograms and summaries are more complex metric types. Not only
+creates a single histogram or summary a multitude of time series, it
```

Comment:
I see, the "do" is missing. http://dictionary.cambridge.org/grammar/british-grammar/not-only-but-also